packages/i18n-calypso/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "i18n-calypso",
 	"version": "4.1.0",
-	"description": "i18n JavaScript library on top of Jed originally used in Calypso",
+	"description": "i18n JavaScript library on top of Tannin originally used in Calypso",
```

Comment:
Noting, there are still many references to Jed in this package's README.

Reply:
Thanks, @aduth! I'll take a look.